### PR TITLE
Fix /reports/<area>/summary dashboard bar graph bugs

### PR DESCRIPTION
**Pull request by zarino**
A bunch of fixes to the horizontal bar charts on `fixmystreet.com/reports/<area>/summary` pages:

* Y-axis labels now wrap onto multiple lines, if they are more than 4 words long. This makes a huge difference on pages like https://www.fixmystreet.com/reports/Oxfordshire/summary where long category names on the Y-axis took up a huge amount of horizontal space.
* Floating value labels are now properly right- or left-aligned, to avoid gaps or overlaps with the ends of the bars.
* Dashboard charts now use the default typeface, rather than the body’s `font-family`, to avoid the situation where the chart would render with invisible text because the custom font file hadn’t loaded yet. (At least, we _think_ that’s what was going on.)

Fixes mysociety/fixmystreet-commercial#969.

### CODE REVIEW

File: web/js/dashboard.js [hunk 1]
```diff
@@ -250,6 +271,8 @@ $(function(){
             }
         });
 
+        window.barChart = barChart;
```
[the remark this answers is not in the document]
Comment (zarino) [reply web/js/dashboard.js]: Gahh! I had it in there for debugging. Will remove.

File: web/js/dashboard.js [hunk 2]
```diff
@@ -53,6 +53,26 @@ $(function(){
...
 
+    // Wraps a row label onto two equal equal lines,
+    // if it is longer than 4 words.
+    var linewrapLabel = function(text) {
+        if ( text.split(' ').length < 5 ) {
+            return text;
+        }
+
+        var middle = Math.floor(text.length / 2);
+        var before = text.lastIndexOf(' ', middle);
```
[the remark this answers is not in the document]
Comment (zarino) [reply web/js/dashboard.js]: I don’t think chart.js works in IE8 either, so… 😕

[the remark this answers is not in the document]
Comment (dracos) [reply web/js/dashboard.js]: Ah that's fine then ;) Guess might want to check our error handling deals with this/that okay...

[the remark this answers is not in the document]
Comment (zarino) [reply web/js/dashboard.js]: Mmm indeed. I’ve not logged into the `/reports/<area>/summary` dashboard, but even just the regular `/reports` dashboard needs some TLC when it comes to Internet Explorer. I'll open up a new ticket for it.

**Pull request merged**